web/modules/custom/portland/modules/portland_glossary/js/glossary-term.js

```diff
@@ -0,0 +1,210 @@
+Drupal.behaviors.dynamicGlossaryTooltip = {
+  attach(context, drupalSettings) {
+    once('dynamicGlossaryTooltip', '[data-entity-substitution="glossary_term"]', context).forEach(link => {
+      const uuid = link.getAttribute('data-entity-uuid');
+      if (!uuid) return;
+
+      const path = `/jsonapi/node/glossary_term/${encodeURIComponent(uuid)}`;
+
+      fetch(path)
+        .then(res => {
+          const contentType = res.headers.get('content-type') || '';
+          if (!res.ok || !contentType.includes('application/json')) {
+            throw new Error(`Invalid JSON response for UUID "${uuid}"`);
+          }
+          return res.json();
+        })
+        .then(data => {
+          if (!data || !Array.isArray(data) || data.length === 0) {
+            throw new Error(`No data found for UUID "${uuid}".`);
+          }
+
+          const termData = data[0];
+          const termLabel = termData.title || 'Glossary Term';
+          const pronunciation = termData.pronunciation || '';
+          const description = termData.short_definition || 'No description available.';
+          const url = termData.url || '#';
+          const hasLongDefinition = !!termData.has_long_definition;
+
+          const seeAlsoLinks = Array.isArray(termData.see_also) && termData.see_also.length
+            ? `<div class="term-see-also"><strong>See also:</strong> ` +
+            termData.see_also.map(item =>
+              `<a href="${item.url}" rel="noopener noreferrer">${item.title}</a>`
```

Comment:
Curious about the "rel" attribute value. Currently the glossary URLs should be all internal. Is this value added to be ready for glossary URLs from third party sites?